### PR TITLE
Add native-java fix to buildifier.

### PR DESCRIPTION
**Pull request by iirina**

### CODE REVIEW

File: WARNINGS.md [hunk 1]
```diff
@@ -468,6 +469,16 @@ disabled](https://github.com/bazelbuild/bazel/issues/8391).
 
 --------------------------------------------------------------------------------
 
+## <a name="native-java"></a>All Java build rules should be loaded from Starlark
+
+  * Category name: `native-java`
```
Comment (vladmos) [inline WARNINGS.md]: Please mention the flag here:

      * Flag in Bazel: [`--incompatible_load_java_rules_from_bzl`](https://github.com/bazelbuild/bazel/issues/8746)

File: WARNINGS.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Warning categories supported by buildifier's linter:
   * [module-docstring](#module-docstring)
   * [name-conventions](#name-conventions)
   * [native-android](#native-android)
+  * [native-java](#native-java)
```
Comment (vladmos) [inline WARNINGS.md]: nit: lexicographically it should be one line below

File: WARNINGS.md [hunk 3]
```diff
@@ -468,6 +469,16 @@ disabled](https://github.com/bazelbuild/bazel/issues/8391).
 
 --------------------------------------------------------------------------------
 
+## <a name="native-java"></a>All Java build rules should be loaded from Starlark
```
Comment (vladmos) [inline WARNINGS.md]: nit: move after `native-build`

File: warn/warn.go [hunk 4]
```diff
@@ -127,6 +127,7 @@ var FileWarningMap = map[string]func(f *build.File) []*LinterFinding{
 	"module-docstring":          moduleDocstringWarning,
 	"name-conventions":          nameConventionsWarning,
 	"native-android":            nativeAndroidRulesWarning,
+	"native-java":               nativeJavaRulesWarning,
```
Comment (vladmos) [inline warn/warn.go]: nit: move below

**Comment by iirina** [comment]
FYI @oquenchil

File: WARNINGS.md (hunk 1 above)
Comment (iirina) [reply WARNINGS.md]: Done.

File: WARNINGS.md (hunk 3 above)
Comment (iirina) [reply WARNINGS.md]: Done.

File: warn/warn.go (hunk 4 above)
Comment (iirina) [reply warn/warn.go]: Done.

File: tables/tables.go [hunk 5]
```diff
@@ -223,6 +223,23 @@ var AndroidNativeRules = []string{
...
+	"java_lite_proto_library",
+	"java_proto_library",
+	"java_test",
+	"java_package_configuration",
+	"java_plugin",
+	"java_runtime",
+	"java_toolchain",
+}
+
+// AndroidLoadPath is the load path for the Starlark Android Rules.
```
Comment (vladmos) [inline tables/tables.go]: Please fix the comment (golint will warn about it).

Comment (iirina) [reply tables/tables.go]: Oh cool. Thanks for catching it. Done.

**Comment by iirina** [comment]
Thanks for the review @vladmos!

**Pull request merged**